recipes/r-codemetar/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set version = '0.1.8' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-codemetar
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/codemetar_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/codemetar/codemetar_{{ version }}.tar.gz
+  sha256: e255091f44c6160586699662acdc231b82a18a5ea9eb10670f509e0e01ec17ff
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-commonmark
+    - r-crul
+    - r-curl
+    - r-desc
+    - r-gh
+    - r-git2r
+    - r-glue
+    - r-jsonlite >=1.6
+    - r-magrittr
+    - r-memoise
+    - r-pingr
+    - r-pkgbuild
+    - r-purrr
+    - r-readr
+    - r-sessioninfo
+    - r-stringi
+    - r-stringr
+    - r-tibble
+    - r-usethis
+    - r-whisker
+    - r-xml2
+  run:
+    - r-base
+    - r-commonmark
+    - r-crul
+    - r-curl
+    - r-desc
+    - r-gh
+    - r-git2r
+    - r-glue
+    - r-jsonlite >=1.6
+    - r-magrittr
+    - r-memoise
+    - r-pingr
+    - r-pkgbuild
+    - r-purrr
+    - r-readr
+    - r-sessioninfo
+    - r-stringi
+    - r-stringr
+    - r-tibble
+    - r-usethis
+    - r-whisker
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('codemetar')"           # [not win]
+    - "\"%R%\" -e \"library('codemetar')\""  # [win]
+
+about:
+  home: https://github.com/ropensci/codemetar, https://ropensci.github.io/codemetar
+  license: GPL-3
+  summary: The 'Codemeta' Project defines a 'JSON-LD' format for describing software metadata,
+    as detailed at <https://codemeta.github.io>. This package provides utilities to
+    generate, parse, and modify 'codemeta.json' files automatically for R packages,
+    as well as tools and examples for working with 'codemeta.json' 'JSON-LD' more generally.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
```

Comment:
shoudl I check if `conda-forge/r` is OK with being a maintainer?

Reply:
No need, we're all in by default ;-)